Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
```

Comment:
```suggestion
- Mild unprofessionalism: see this [guide](https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php)
```
Minor typo breaking list + improve link formatting

Reply:
In relation to @rlmoser99 comment below about sending users to this external link, if it *is* decided to keep this link, the verbiage/link text I would say should instead be, `see this guide on [Group Chat Etiquette in a Professional Environment]`